packages/acceptance-tests/pkg-tests-specs/sources/plugins/plugin-typescript.test.ts

```diff
@@ -44,6 +44,66 @@ describe(`Plugins`, () => {
         }),
       );
 
+      test(
+        `it should automatically enable automatic @types insertion when a tsconfig is detected in the current workspace`,
+        makeTemporaryMonorepoEnv({
+          workspaces: [`packages/*`],
+        }, {[`packages/foo`]: {}}, async ({path, run, source}) => {
+          await xfs.writeFilePromise(ppath.join(path, `packages/foo/tsconfig.json`), ``);
+
+          await run(`add`, `is-number`, {
+            cwd: `${path}/packages/foo` as PortablePath,
+          });
+
+          await expect(readManifest(`${path}/packages/foo` as PortablePath)).resolves.toMatchObject({
+            dependencies: {
+              [`is-number`]: `^2.0.0`,
+            },
+            devDependencies: {
+              [`@types/is-number`]: `^2`,
+            },
+          });
+        }),
+      );
+
+      test(
+        `it should not automatically enable automatic @types insertion when a tsconfig is present in a sibling workspace`,
+        makeTemporaryMonorepoEnv({
+          workspaces: [`packages/*`],
+        }, {[`packages/foo`]: {}, [`packages/bar`]: {}}, async ({path, run, source}) => {
+          await xfs.writeFilePromise(ppath.join(path, `packages/foo/tsconfig.json`), ``);
+
+          await run(`add`, `is-number`, {
+            cwd: `${path}/packages/bar` as PortablePath,
+          });
+
+          await expect(readManifest(`${path}/packages/bar` as PortablePath)).resolves.toMatchObject({
+            dependencies: {
+              [`is-number`]: `^2.0.0`,
+            },
+          });
+        }),
+      );
+
+      test(
+        `it should not automatically enable automatic @types insertion when a tsconfig is detected in the root project of the current workspace`,
```

Comment:
I'm a little torn on this behavior. If I have a `tsconfig.json` at the root of my monorepo, but not in my current workspace, I think it's logical not to expect the plugin to be auto enabled. On the other hand, it gets a little harder to document/explain and might seem like a bug. Thoughts?